Makefile

```diff
@@ -47,3 +47,8 @@ tc-worker-env:
 tc-worker:
 	CGO_ENABLED=$(CGO_ENABLE) GOARCH=amd64 go build
 	docker build -t taskcluster/tc-worker -f tc-worker.Dockerfile .
+
+lint:
+	go get github.com/alecthomas/gometalinter
+	gometalinter --install
+	$(GOPATH)/bin/gometalinter --deadline 10m $$(go list ./... | grep -v /vendor/ | sed 's/github.com\/taskcluster\/taskcluster-worker/./')
```

Comment:
If we use an include list things will break when metalinter adds new checks... But yes, I like the idea of adding more checks :)

But please can we do it one at the time... And briefly discuss it.. some of the ones that complains about code duplication are tempting to ignore... Code duplication in tests is sometimes the simplest, because we do things with little variations...